shell/docker-builder.sh

```diff
@@ -28,13 +28,14 @@ args=("--ssh" "default" "--progress=plain" "--file" "deployments/${appName}/Dock
 
 # Build a quick native image on PRs and load it into docker cache
 # for security scanning
-if [[ -z $CIRCLE_TAG ]]; then
+# TODO: uncomment
```

Comment:
I will uncomment it before final merge :), this is now needed to simulate tagged circle ci run.